README.md

```diff
@@ -1,36 +1,31 @@
-[![Manager](https://github.com/thoughtworks/maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/manager.yml)
-[![Gateway](https://github.com/thoughtworks/maeve-csms/workflows/Gateway/badge.svg)](https://github.com/thoughtworks/maeve-csms/actions/workflows/gateway.yml)
+[![Manager](https://github.com/thoughtworks/Maeve-csms/workflows/Manager/badge.svg)](https://github.com/thoughtworks/Maeve-csms/actions/workflows/manager.yml)
```

Comment:
The capitalization change breaks these links.

Reply:
The URLs still resolve for me